infrastructure/cluster/flux/rucio-cronjobs.yaml

```diff
@@ -79,7 +79,7 @@ spec:
               command:
               - /bin/sh
               - -c
-              - date; pwd; cd /opt/rucio/etc/; ls; echo Hello from rucio-noise container; export RUCIO_CONFIG=/opt/rucio/etc/rucio-sso.cfg; echo Exported config; rucio -vvv whoami; FILE_SIZE=10M /bin/bash /home/user/vre/containers/rucio-noise/produce_noise.sh; echo 'DB cleanup Done!'
+              - date; pwd; cd /opt/rucio/etc/; ls; echo Hello from rucio-noise container; export RUCIO_CONFIG=/opt/rucio/etc/rucio-sso.cfg; echo Exported config; rucio -vvv whoami; export FSIZE=10M; FILE_SIZE=${FSIZE} /bin/bash /home/user/vre/infrastructure/scripts/cron_scripts/rucio-noise/produce_noise.sh; echo 'Rucio noise ${FSIZE} Done!'
```

Comment:
I didn't fully get it Domenic,
If we don't install the package (what I would completely support), the only option that exists is cloning the repo. If we do so, we'd need to move directly to the relative path (locally inside the container), so I don't understand what you are mentioning. 
Also, if we are keeping the scripts that are run within a container in the same folder that builds the container, where should we put it ?

Reply:
The part `/home/user/vre/` is your local path no? I should be relative to the `Dockerfile`?

Reply:
When building the image, we are not installing any package, just cloning the `vre` repo. Later, when the cronjob will be executed, and because the repository was cloned and NOT installed, the script will be found in the dir relative to where the repo was clone. You are indeed running the script INSIDE the container, that it's being executed. Thus the `dockerfile` has nothing to do in this case.

In other words, if ever the repo was cloned in a different location than `/home/user/`, to be able to run the script you'd need to point the new location.